ansible_collections/arista/avd/roles/eos_config_deploy_cvp/README.md

```diff
@@ -127,7 +127,7 @@ tasks:
 
 ### Ignore devices not provisioned in Cloudvision
 
-When you want to provision a complete topoplogy and devices are not already in Cloudvision, you can configure inventory to ignore these devices by using a host variable: `is_deployed`
+When you want to provision a complete topology and devices are not already in Cloudvision, you can configure inventory to ignore these devices by using a host variable: `is_deployed`. This will also cause the ports facing the undeployed device to be shutdown in order to satisfy the validate role when checking if a port should be enabled but is down due to the other device not being present.
```

Comment:
This documentation is only for the `eos_config_deploy_cvp` role, so we should not document the `eos_designs` behavior here. Please add documentation for this knob under `eos_designs` `Fabric Variables` as well as a new knob `is_deployed_shutdown: < true | false | default -> false >` and implement check for this new knob before you enable this new behavior. Make sure it is clear that the `is_deployed` should be a host_var and `is_deployed_shutdown` can be a group_var.

It is a nice idea, but I would consider this a breaking change for existing deployments, hence the feature knob. If you have better ideas for naming you can implement that.

Reply:
Agreed, I'll make the changes and re push. Thanks.

Reply:
Created `is_deployed_shutdown_peer_interfaces` a fabric level variable as I don't think it makes sense for this to be group level as it would be easy to forget to enable each time a new group is added.

Documented this toggle under `eos_designs` and updated `eos_config_deploy_cvp` docs to make it clear the host level `is_deployed: false` toggle will only affect actual device configs when fabric level `is_deployed_shutdown_peer_interfaces: true`. Host level `is_deployed: false` will still cause the device to be ignored in CVP.